packages/ir-sdk/fern/apis/ir-types-v38/changelog/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [v38] - 2024-04-17
+
+- Feature: Add support for Server-Sent-Events to Streaming HTTP Responses
+  Read more about SSE here: https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events. 
```

Comment:
@amckinney would you be up to bring this back up and add ir v39 here?

Reply:
eventual plan is to add this to the docs

Reply:
I added a new `CHANGELOG.md` under `ir-types-latest`, I just mistakenly copied this over to `ir-types-v38` instead of merging them together 🤦 I'll update here!